dpipe/predict/shape.py

```diff
@@ -97,6 +97,9 @@ def patches_grid(patch_size: AxesLike, stride: AxesLike, axis: AxesLike = None,
     """
     valid = padding_values is not None
 
+    if stream and async_predict:
+        raise ValueError('Unable to use async_predict when using stream. Please specify async_predict=False explicitly')
```

Comment:
My bad, for now lets just throw warning that `async_predict` has no effect in presence of `stream`